rules/linux/execution_python_tty_shell.toml

```diff
@@ -27,7 +27,10 @@ type = "eql"
 
 query = '''
 sequence with maxspan=1m
-  [process where host.os.type == "linux" and event.type == "start" and process.name : "python*"] by process.entity_id
+  [process where host.os.type == "linux" and event.type == "start" and process.name : "python*" and
+   not (process.args == "/bin/yum" and process.args == "update") and 
```

Comment:
This would eliminate one of the few possible false positives, please use the sample tuned query pinned in the issue to test on your work load.